src/httpd_jsonapi.c

```diff
@@ -1830,6 +1944,13 @@ jsonapi_reply_queue_tracks_add(struct httpd_request *hreq)
   if (ret < 0)
     return HTTP_INTERNAL;
 
+  // If query parameter "play" is "true", start playback after successfully adding new items
```

Comment:
play=true sounds a bit odd to me, also taking into account that play=false wouldn't do the opposite. You could just look for "play" without any value? Or if you want to prepare for more playback control then maybe playback=start.